src/App.tsx

```diff
@@ -0,0 +1,9 @@
+const App = (): JSX.Element => {
+  return (
+    <div className="App">
+      <h1>Dyson Front-end React Template Project</h1>
+    </div>
+  );
+};
+
+export default App;
```

Comment:
Why we have this approach to define variable when not needed? I'm not saying it's wrong. I just don't understand it. 
```suggestion
export default function App(): JSX.Element {
  return (
    <div className="App">
      <h1>Dyson Front-end React Template Project</h1>
    </div>
  );
};
```
https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export

Reply:
It comes from React & Typescript Cheat Sheet Recommended Practices.
https://github.com/typescript-cheatsheets/react/blob/main/README.md#function-components